### PR TITLE
Merge milestone-5 with development

### PR DESCRIPTION
**Pull request by rociac**
This branch contains all the component interaction and fixed some logic issues. Looking forward for your feedback!

### CODE REVIEW

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi @rociac good job working on this project everything has come along well so far just one small bug left the division button does not work properly it even shows up in the display box unlike the other operators 
[see this](https://www.loom.com/share/992cc362e6f249a9bf85a567be168848)

**please fix that and request for another review**  :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

**Comment by rociac** [comment]
Fixed the typo :)

File: src/logic/operate.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+      return num1.minus(num2).toString();
     case 'X':
-      return num1.mul(num2);
-    case '/':
+      return num1.mul(num2).toString();
+    case '÷':
       if (num2 === 0) return "Can't divide by zero";
-      return num1.div(num2);
+      return num1.div(num2).toString();
     case '%':
```
Comment (shloch) [inline src/logic/operate.js]: i see that you managed division by zero....it still doesn't work😊😊 , Try to debug this .
Try to follow the execution flow between `operate.js` and  `calculate.js`

Screen Shot 2020-02-13 at 6 26 01 PM

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @rociac 

We not quite there yet,
Please look at the issue below

File: src/logic/operate.js (hunk 1 above)
Comment (rociac) [reply src/logic/operate.js]: Fixed it, the problem was using === it was comparing an object with a number. Used .eq method on the big.js module

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED

Hi @rociac this looks good now, everything works fine, please merge and move forward, **this is approved** :+1: 

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**